app/src/androidTest/java/com/google/jetpackcamera/VideoRecordingDeviceTest.kt

```diff
@@ -104,6 +155,23 @@ internal class VideoRecordingDeviceTest {
         )
     }
 
+    private fun deleteFilesInDirAfterTimestamp(timeStamp: Long): Boolean {
+        var hasDeletedFile = false
+        for (file in File(ImageCaptureDeviceTest.DIR_PATH).listFiles()) {
+            if (file.lastModified() >= timeStamp) {
+                file.delete()
+                if (file.exists()) {
+                    file.getCanonicalFile().delete()
+                    if (file.exists()) {
+                        instrumentation.targetContext.applicationContext.deleteFile(file.getName())
+                    }
+                }
+                hasDeletedFile = true
+            }
+        }
+        return hasDeletedFile
+    }
+
     companion object {
         private val TEST_CONTRACT = object : ActivityResultContracts.TakePicture() {
```

Comment:
Can you rename this to IMAGE_TEST_CONTRACT?

Reply:
No longer valid